opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -374,6 +389,8 @@ def _extract_attributes(
     invalid_value_dropped_count = 0
     for key, value in attrs.items() if attrs else []:
         key = _truncate_str(key, 128)[0]
+        if key in LABELS_MAPPING:
+            key = LABELS_MAPPING.get(key, key)
```

Comment:
nit covered by if statement already
```suggestion
            key = LABELS_MAPPING[key]
```

Reply:
yep corrected, but now CI failed, I added that ugly get in the 11st place because of it, not familiar with pylint but surely one of theses cases where we could ignore the linter